content/docs/06.enterprise/secrets-manager.md

```diff
@@ -81,7 +81,7 @@ To leverage [Google Secret Manager](https://cloud.google.com/secret-manager) as
 kestra:
   secret:
     type: google-secret-manager
-    google-secret-manager:
+    googleSecretManager:
       project: gcp-project-id
```

Comment:
I'm curious if this should be projectId based on other google config. I'll check